styles/components/_listingCard.scss

```diff
@@ -237,14 +237,18 @@
   }
 
   .listingsGridListView & {
-    // This isn't ideal, but there's inconistent spacing / widths of cols, plus there's variable
-    // content width on the price (e.g. some currencies may have quite large numbers)
+    // The price column is a set width and the free shipping column is a maximum of 200px and 4 lines.
+    // This leaves the title/description/rating to flex to take up the remaining space.
     .priceCol {
       padding-right: 20px;
+      width: 100px;
+      text-align: right;
     }
 
     .freeShipCol {
       padding-right: 20px;
```

Comment:
Can we remove the right padding? It doesn't seem necessary.

Reply:
Is the right padding still needed if the price gets very large? We don't want the text of the price to touch the free shipping green background.

Maybe change it to 10px?

Reply:
> Is the right padding still needed if the price gets very large?

No. There's a `gutterHSm` class in place which provides a 10px margin. The padding IMHO is superfluous.

![image](https://user-images.githubusercontent.com/794517/40794240-70f86a98-64b3-11e8-9df9-51bfa69361f9.png)

Reply:
(unless you're thinking 10px is too tight, in which case I suppose we could drop the padding to 10, which would result in 20px of space.)

Reply:
Just waiting on the consensus from @jjeffryes when ya get a moment. Would you still like me to remove the padding-right:20px on priceCol?

Reply:
The gutterHSm gives it a 5px margin, I tested a variety of numbers for the price, I think it works fine with the padding-right removed. Please go ahead and remove that.

Reply:
Removed. Ready for merge.